### DISCUSSION TITLE
Initialization of Rancher Desktop fails with "standard user"

### DISCUSSION THREAD

**Discussion by GhuulCZ** [rancher-sandbox/rancher-desktop#6797] [Q&A]
Hello,
I'm trying to evaluate Rancher Desktop under Windows when using BeyondTrust security tool - the BT app acts like authorization "MITM" (basically replaces UAC prompt) that checks what user is trying to run and evaluates whether user can run some administrative commands when allowed or can't run other even as standard user when such commands are prohibited. 

It seems that everything goes well - the RD is installed under standard user privileges, WSL is registered and running, docker engine started - until the "Initialization of Rancher Desktop" where the RD is kind of stuck and I see the repeating messages in the `rancher-desktop-guestagent.log`: 
```
2024/04/30 10:35:28 [DEBUG]   kubernetes: failed to read kubeconfig [config-path=/etc/rancher/k3s/k3s.yaml][error=could not load Kubernetes client config from /etc/rancher/k3s/k3s.yaml: stat /etc/rancher/k3s/k3s.yaml: no such file or directory]
2024/04/30 10:35:29 [DEBUG]   kubernetes: failed to read kubeconfig [config-path=/etc/rancher/k3s/k3s.yaml][error=could not load Kubernetes client config from /etc/rancher/k3s/k3s.yaml: stat /etc/rancher/k3s/k3s.yaml: no such file or directory]
2024/04/30 10:35:30 [DEBUG]   kubernetes: failed to read kubeconfig [config-path=/etc/rancher/k3s/k3s.yaml][error=could not load Kubernetes client config from /etc/rancher/k3s/k3s.yaml: stat /etc/rancher/k3s/k3s.yaml: no such file or directory]
2024/04/30 10:35:31 [DEBUG]   kubernetes: failed to read kubeconfig [error=could not load Kubernetes client config from /etc/rancher/k3s/k3s.yaml: stat /etc/rancher/k3s/k3s.yaml: no such file or directory][config-path=/etc/rancher/k3s/k3s.yaml]
```

I'm trying to find out what happens in this stage and if I'm able to invoke some of the RD tasks manually from eg powershell/cmd to get more more details and see if some of the tasks fail or just hang. Since none of the processes and commands I see in the BT logs are blocked (it may not capture everything though) I wonder whether the problem is BT itself blocking something we do not see or simply the lack of admin rights.

Can somebody push me please to the right direction?

Thanks 
Pavel

**Comment by cobantun** [comment]
I'm seeing the the same log
